README.md

```diff
@@ -141,6 +149,17 @@ The default value is `zlib.Z_DEFAULT_WINDOWBITS`, or `15`.
 See [Node.js documentation](http://nodejs.org/api/zlib.html#zlib_memory_usage_tuning)
 regarding the usage.
 
+##### params [*(brotli only)*](https://nodejs.org/api/zlib.html#zlib_brotli_constants)
+
+  - `zlib.constants.BROTLI_PARAM_MODE`
```

Comment:
It's quite unclear from this documentation what `params` is supposed to take as a value... is this a list or something? Can you add a plain description around this option and what it does, etc. like the other options?

Reply:
The `params` are extra parameters object. That's how the zlib module is defined in node.
Anyone playing with the parameters is actually using zlib options, and is either familiar with it or going to read the docs. There's no point in recreating node's docs in here.

Reply:
Yes, there is a point. The reason they are recreated is because I had the same attitude, but after the x-th issue opened in this repo asking the same question, it is worth recreating all the docs here. If you disagree, you're always welcome to host, maintain, and triage incoming issues on your own module :) Please write out all the docs for all options that this module is exposing directly in the README.